log/log.go

```diff
@@ -45,6 +48,31 @@ func NewLogger(config config.Log) (*zap.Logger, error) {
 	return logger, nil
 }
 
+func StreamAccessLogHandler(srv interface{}, ss grpc.ServerStream, info *grpc.StreamServerInfo, handler grpc.StreamHandler) error {
```

Comment:
__[golint]__
* [golint] exported function StreamAccessLogHandler should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->